AWSCoreTests/AWSTestUtility.m

```diff
@@ -65,6 +65,57 @@ + (void)initialize {
     [super initialize];
 }
 
++ (void) setupSessionCredentialsProvider {
+    
+    NSDictionary *testConfigurationJson = [self getTestConfigurationJSON];
+    AWSBasicSessionCredentialsProvider *credentialsProvider = [[AWSBasicSessionCredentialsProvider alloc]
+                                                               initWithAccessKey:testConfigurationJson[@"Credentials"][@"accessKey"]
```

Comment:
As much as you're referencing `testConfigurationJson[@"Credentials"]` in this file, it would be nice to make a helper method for it

Reply:
done